.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,37 @@
+<!-- Write all of the issues that are linked to this pull request. -->
+<!-- If this PR is enough to close them you can write something like "Closes #314 and closes #42" -->
+<!-- If you just want to reference them without closing them, you can add something like "References #112" -->
 Closes #
 
-## Proposed Changes
+<!-- Add a short description of the PR content here-->
+
 
+## Proposed Changes
+<!-- List here major points of changes, so that the reviewers can have a bit more context while looking at your work! -->
   -
   -
-  -
\ No newline at end of file
+  -
+
+## Change Type
+<!-- Indicate the type of change you think your pull request is -->
+- [ ] `bugfix` (+0.0.1)
+- [ ] `minor` (+0.1.0)
+- [ ] `major`  (+1.0.0)
+- [ ] `internal` (no version update)
+- [ ] `documentation` (no version update)
+
+## Checklist before review
+<!-- If this section is not clear, please read this part of the docs: https://phys2bids.readthedocs.io/en/latest/contributorfile.html#pr -->
+<!-- You're invited to open a draft PR ASAP, but before marking it "ready for review", check that you have done the following: -->
+- [ ] I added everything I wanted to add to this PR.
+- [ ] \[Code or tests only\] I wrote/updated the necessary docstrings.
+- [ ] \[Code or tests only\] I ran and passed tests locally.
+- [ ] \[Documentation only\] I built the docs locally.
+- [ ] My contribution is harmonious with the rest of the code: I'm not introducing repetitions.
+- [ ] My code respects the adopted style, especially linting conventions.
+- [ ] The title of this PR is explanatory on its own, enough to be understood as part of a changelog.
+- [ ] I added or indicated the right labels.
+<!-- If relevant, you can add a milestone label or indicate an ideal timeline for this PR, including whether the progress of this PR is linked to other PRs being completed before or after it. -->
+- [ ] I added information regarding the timeline of completion for this PR.
+<!-- If you want, you can ask for reviews on a draft PR -->
+- [ ] Please, comment on my PR while it's a draft and give me feedback on the development!
```

Comment:
Shouldn't this last sentence be for the reviewers?

Reply:
Ah, no. This is for the user to clarify that they want a review while drafting (otherwise in the documentation we wrote not to do that)